exec/cnex/object.h

```diff
@@ -21,7 +22,9 @@ typedef enum tagNObjectType {
 typedef struct tagTObject {
     ObjectType      type;
     int             refcount;
-    struct tagTCell *cell;
+    void            *cell;
```

Comment:
If this isn't going to always be a TCell, then it shouldn't be called `cell`.

Reply:
Changed it to `ptr`.  Considered `objPtr`, but decided to go with the shorter name.